test/server/index.test.js

```diff
@@ -0,0 +1,190 @@
+const assert = require('assert');
+const supertest = require('supertest');
+const server = supertest.agent('http://localhost:8080');
+
+describe('missingEmail error test', function() {
+  it('Signup success', function(done) {
+    server
+      .post('/api/signup')
+      .send({'username': 'aliceShen12345',
```

Comment:
Just fyi, you dont need to deal with this now.
In this case, you are working with real data and using the real database. actually in real tests we never do this. We [mock](https://en.wikipedia.org/wiki/Mock_object) all of the dependencies and using so called dummy data.